android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt

```diff
@@ -399,11 +407,24 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
             }
         }
 
+        val offlineBehaviorParam = if (amount > offlineModeTransactionLimit
+            || (terminal.offlineStatus.sdk.offlinePaymentAmountsByCurrency[currency]?.toInt() ?: 0) > offlineModeStoredTransactionLimit
+        ) { OfflineBehavior.REQUIRE_ONLINE } else {
```

Comment:
same comment as iOS, we can remove this and just directly use the OfflineBehavior passes by the integration